README.md

```diff
@@ -1,7 +1,7 @@
 # Vuetify Date Range Picker
 
 > The missing date range picker for Vuetify JS you have been looking for.
-> ![Date Range Picker](https://github.com/praveenpuglia/vuetify-daterange-picker/blob/master/public/updated-date-range-lres.gif)
+> ![Date Range Picker](https://raw.githubusercontent.com/praveenpuglia/vuetify-daterange-picker/1eb3ba1d/public/updated-date-range-lres.gif)
```

Comment:
In NPM `.gif` is not rendering so changed link using https://rawgit.com/

Reply:
Noticed that. Thanks 👍